test/integration/presence_test.go

```diff
@@ -163,23 +179,19 @@ func TestPresence(t *testing.T) {
 							Presences: wr.Presences,
 						})
 					}
-					if len(responsePairs) == 3 {
+					if len(responsePairs) == 1 {
 						return
 					}
 				}
 			}
 		}()
 
 		// 03. Watch the second client's document.
-		expected = append(expected, watchResponsePair{
```

Comment:
Any reason for removing `expected` code and changing `len(responsePairs) == 3` to `len(responsePairs) == 1`?